rest/taskana-rest-spring/src/main/java/pro/taskana/common/rest/InitBinderControllerAdvice.java

```diff
@@ -27,5 +28,7 @@ public void initBinder(WebDataBinder binder) {
         new JsonPropertyEditor(objectMapper, PriorityColumnHeaderRepresentationModel.class));
     binder.registerCustomEditor(
         ObjectReference.class, new JsonPropertyEditor(objectMapper, ObjectReferenceImpl.class));
+    // https://taskana.atlassian.net/browse/TSK-1699
```

Comment:
Personally I am not a fan of such comments since I can see the commit you've edited that line through git. 
See:
![Screenshot from 2023-01-17 08-47-16](https://user-images.githubusercontent.com/15628173/212839277-dac33cce-a544-474a-9c84-a9c3a974c946.png)